bridge-filtering

```diff
@@ -0,0 +1,313 @@
+#!/bin/bash
+
+set -euo pipefail
+
+stdin=$(cat /dev/stdin)
+logFile="${LOGFILE:-/var/log/bridge-filtering.log}"
+
+KUBECONFIG_PATH="${KUBECONFIG_PATH:-/etc/cni/net.d/bridge-filtering.d/bridge-filtering.kubeconfig}"
+CNI_VERSION="$(echo "$stdin" | jq -r ".cniVersion")"
+CNI_PREV_RESULT="$(echo "$stdin" | jq -cr ".prevResult")"
+
+exec 2>> $logFile
+
+NFT_BRIDGE_TABLE=bridge
+NFT_TABLE=filter
+
+NFT_INGRESS_CHAIN=prerouting
+NFT_INGRESS_HOOK=prerouting
+
+NFT_POSTROUTING_CHAIN=postrouting
+NFT_EGRESS_HOOK=postrouting
+
+get_object() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -cr "$json_path"
+}
+
+get_array_items() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -c $json_path | jq -cr ".[]"
+}
+
+get_array_len() {
+    local json_object="$1"
+    local json_path="$2"
+    echo "$json_object" | jq -c "$json_path" | jq ". | length"
+}
+
+for_json_array() {
+    local json_object="$1"
+    local json_path="$2"
+    local fn="$3"
+    for item in $(get_array_items "$json_object" "$json_path"); do
+        $fn $item
+    done
+}
+
+get_ip_version() {
+    local ip_address="$1"
+    if [[ "$ip_address" =~ .*:.* ]]
+    then
```

Comment:
I think better to have the then line above please

Reply:
[Done](https://github.com/rhrazdil/cidr-filtering-cni/pull/13/commits/c03eb9b9294e1922674c685b4e99cec2df2d5055)